portfolio/src/main/java/com/google/sps/servlets/servlets/OAuthServlet.java

```diff
@@ -52,10 +52,10 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     //add credentials to datastore (just do this once for deployment)
```

Comment:
comments should have proper punctuation (space between / and start of sentence, capitalization).